dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataMaxRowsParser.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2022 Google LLC
+ * Copyright 2013-2021 CompilerWorks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.edwmigration.dumper.application.dumper.connector.teradata;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static com.google.common.collect.ImmutableMap.toImmutableMap;
+import static java.util.Arrays.stream;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+import com.google.edwmigration.dumper.application.dumper.ArgumentValidationException;
+import com.google.edwmigration.dumper.application.dumper.ConnectorArguments;
+import java.util.List;
+import java.util.Map;
+
+public class TeradataMaxRowsParser {
```

Comment:
Consider making it final and adding a javadoc

Reply:
Done.